plugins/orchestrator-backend/src/service/WorkflowService.ts

```diff
@@ -137,23 +163,31 @@ export class WorkflowService {
   }
 
   async deleteWorkflowDefinitionById(uri: string): Promise<void> {
-    const definitionsPath = this.resolveRosourcePath(uri);
+    const definitionsPath = this.resolveResourcePath(uri);
     await fs.rm(definitionsPath, { force: true });
   }
 
   async listStoredSpecs(): Promise<WorkflowSpecFile[]> {
     const specs: WorkflowSpecFile[] = [];
     // We can list all spec files from FS but let's keep it simple for now
     for (const relativePath of spec_files) {
-      const path = this.resolveRosourcePath(relativePath);
+      const path = this.resolveResourcePath(relativePath);
       const buffer = await fs.readFile(path);
       const content = JSON.parse(buffer.toString('utf8'));
       specs.push({ path, content });
     }
     return specs;
   }
 
-  private resolveRosourcePath(relativePath: string): string {
-    return resolve(join(this.sonataFlowResourcesPath, relativePath));
+  private resolveResourcePath(relativePath: string): string {
+    return resolve(
+      join(this.sonataFlowResourcesPath, default_workflows_path, relativePath),
+    );
+  }
+  async reloadWorkflows() {
+    this.logger.info(`Reloading workflows from Git`);
+    const localPath = `${this.sonataFlowResourcesPath}`;
+    await fs.remove(localPath);
```

Comment:
```suggestion
    await fs.remove(this.sonataFlowResourcesPath);
```
This can be simplified.